win32/devdoc/threadpool_win32.md

```diff
@@ -215,7 +215,7 @@ MOCKABLE_FUNCTION(, int, threadpool_timer_restart, TIMER_INSTANCE_HANDLE, timer,
 
 **SRS_THREADPOOL_WIN32_42_019: [** If `timer` is `NULL`, `threadpool_timer_restart` shall fail and return a non-zero value. **]**
 
-If timer state is `DESTROYING`, `threadpool_timer_restart` shall fail and return a non-zero value.
+**SRS_THREADPOOL_WIN32_07_001: [** If timer state is in destroying state, `threadpool_timer_restart` shall fail and return a non-zero value. **]**
```

Comment:
>destroying [](http://example.com/codeflow?start=56&length=10)

I think capital and enclosed by tickmarks was better.

Reply:
Because they are references to states that should be defined in the doc.

Reply:
done